### PR TITLE
[GUI] Menu Bar/Tool bar (New)

### LINKED ISSUE #39: [GUI] Menu Bar/Tool bar
Menu bar implementation need by @JeewakaSankalpa 

- New entry
- Save

### PR DESCRIPTION
**Pull request by JeewakaSankalpa**
Something went wrong with the last PR
- [x] closes #39
- [ ] tests added / passed
- [ ] passes ``git diff upstream/master | flake8 --diff``

### CODE REVIEW

**Comment by jaehoonhwang** [comment]
I know what it is.
While I fix it please do the presentation slides.

**Comment by jaehoonhwang** [comment]
Resolve the conflict and you shouldn't have any problem.

File: captslog/ui/main.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 from __future__ import absolute_import, unicode_literals
-from PyQt4 import QtCore, QtGui
+from PySide import QtCore, QtGui
```
Comment (jaehoonhwang) [inline captslog/ui/main.py]: Remove this ?

Comment (JeewakaSankalpa) [inline captslog/ui/main.py]: PyQt does not work for me for some reason.  What do I need to do to fix?

Comment (jaehoonhwang) [inline captslog/ui/main.py]: I believe `PyQt4` work in order for `PySide` to work..?

Comment (jaehoonhwang) [inline captslog/ui/main.py]: I'll merge this after we are done with presentation, but for the presentation, keep your branch alive.

Comment (JeewakaSankalpa) [inline captslog/ui/main.py]: yes but when I use pyQT instead of pySide it gives an error

Comment (jaehoonhwang) [inline captslog/ui/main.py]: Open `cmd`.
Type `python`
Inside of python running environment, type `from PyQt4 import QtCore`

**Comment by jaehoonhwang** [comment]
You gottttta resolve the conflict before I can merge

**Comment by jaehoonhwang** [comment]
Also can you add .idea to gitignore?
'.idea'

**Comment by jaehoonhwang** [comment]
You jist need to make an entry in a new line

**Comment by jaehoonhwang** [comment]
You gotta exclude that .idea file please help.

**Comment by jaehoonhwang** [comment]
delete .idea file

**Pull request merged**